src/main/java/org/kuali/kra/protocol/onlinereview/ProtocolOnlineReviewLookupableHelperServiceImplBase.java

```diff
@@ -222,4 +223,27 @@ private AnchorHtmlData getAnchorHtmlData(Properties parameters, String displayTe
         String href  = UrlFactory.parameterizeUrl("../" + getHtmlAction(), parameters);
         return new AnchorHtmlData(href, KRADConstants.DOC_HANDLER_METHOD, displayText);
     }
+
+    @Override
+    public void performClear(LookupForm lookupForm) {
+        for (Iterator iter = this.getRows().iterator(); iter.hasNext();) {
+            Row row = (Row) iter.next();
+            for (Iterator iterator = row.getFields().iterator(); iterator.hasNext();) {
+                Field field = (Field) iterator.next();
+                if (field.isSecure()) {
+                    field.setSecure(false);
+                    field.setDisplayMaskValue(null);
+                    field.setEncryptedValue(null);
+                }
+
+                if ((!field.getFieldType().equals(Field.RADIO)) && (!field.getPropertyName().equals("lookupReviewerPersonId"))) {
```

Comment:
Please create a String constant in the class with the name reviewerPersonIdFieldName="lookupReviewerPersonId" and use that constant on line 239 instead of the direct name.

Reply:
I used the already existing string constant 'REVIEWER_EMPLOYEE' for value of 'lookupReviewerPersonId' in the if statement. 
Thanks